src/Bot.ts

```diff
@@ -1,44 +1,69 @@
 "use strict";
 
+import { calculateBotPick } from "mtgdraftbots";
+
 import { Card } from "./Cards";
 
+const basicOracleIds = [
+	"56719f6a-1a6c-4c0a-8d21-18f7d7350b68",
+	"b2c6aa39-2d2a-459c-a555-fb48ba993373",
+	"bc71ebf6-2056-41f7-be35-b2e5c34afa99",
+	"b34bb2dc-c1af-4d77-b0b3-a0fb342a5fc6",
+	"a3fb7228-e76b-4e96-a40e-20b5fed75685",
+];
+
 export default class Bot {
 	name: string;
 	id: string;
-	cards: Card[] = []; // For debugging mostly.
-	pickedColors: { [color: string]: number } = { W: 0, U: 0, R: 0, B: 0, G: 0 };
+	oracleIds: string[] = [];
+	seen: number[] = [];
+	picked: number[] = [];
+	// This has to be tracked separately since it is used otuside the class.
+	cards: Card[] = [];
 
 	constructor(name: string, id: string) {
 		this.name = name;
 		this.id = id; // Used for sorting
 	}
 
-	pick(booster: Card[]) {
-		let maxScore = 0;
-		let bestPick = 0;
-		for (let idx = 0; idx < booster.length; ++idx) {
-			let c = booster[idx];
-			// TODO: Rate cards
-			let score = c.rating;
-			for (let color of c.colors) {
-				score += 0.35 * this.pickedColors[color];
-			}
-			if (score > maxScore) {
-				maxScore = score;
-				bestPick = idx;
-			}
-		}
-		for (let color of booster[bestPick].colors) {
-			this.pickedColors[color] += 1;
-		}
+	getBotResult(booster: Card[], boosterNum: number, numBoosters: number, pickNum: number, numPicks: number) {
+		const boosterIdxs = booster.map((_, idx) => idx + this.oracleIds.length);
+		this.seen.push(...boosterIdxs);
+		this.oracleIds.push(...booster.map(({ oracle_id }) => oracle_id));
+		const drafterState = {
+			basics: [0, 1, 2, 3, 4],
+			cardsInPack: booster.map((_, idx) => idx + 5),
+			picked: this.cards.map((_, idx) => idx + 5 + booster.length),
+			seen: this.seen.map((_, idx) => idx + 5 + booster.length + this.cards.length),
```

Comment:
```suggestion
			cardsInPack: boosterIdxs,
			picked: this.picked,
			seen: this.seen,
```
The five basics are already accounted for by `this.oracleIds.length`

Reply:
Yeah I forgot to include them and apparently to update the drafterstate object with the correct values when I updated everything else. Fixed now.